src/generator/AutoRest.Go/Model/MethodGo.cs

```diff
@@ -41,6 +43,18 @@ internal void Transform(CodeModelGo cmg)
       PackageName = cmg.Namespace;
       NextAlreadyDefined = NextMethodExists(cmg.Methods.Cast<MethodGo>());
 
+      var apiVersionParam =
+        from p in Parameters
+        let name = p.SerializedName.Value
+        where name != null && name.IsApiVersion()
+        select p.DefaultValue.Value?.Trim(new[]{'"'});
+
+      APIVersion = apiVersionParam.SingleOrDefault();
+      if(APIVersion == default(string))
```

Comment:
Ahhh, so I hit at least one case where the APIVersion in the parameter list was the empty string. In that case, we should default back to the APIVersion defined at package level. Perhaps the above isn't super readable, how about this:

``` C#
if apiVersionParam.Any()
{
    APIVersion = apiVerisonParam.Single();
}
else
{
    APIVersion = cmg.ApiVersion;
}
```